BUILD

```diff
@@ -1,35 +1,20 @@
 # gRPC Bazel BUILD file.
 #
-# Copyright 2016, Google Inc.
-# All rights reserved.
+# Copyright 2016 gRPC authors.
```

Comment:
Should "gRPC Authors" be capitalized? That's how Java files look today (except for its LICENSE file)